local_planner/include/local_planner/local_planner.h

```diff
@@ -52,6 +52,7 @@ class LocalPlanner {
   float min_realsense_dist_ = 0.2f;
   float smoothing_margin_degrees_ = 30.f;
   float max_point_age_s_ = 10;
+  float pos_to_goal_dist_xy_ = 100.f;
```

Comment:
can you call it `dist_to_goal_xy_`? Since `pos` seems to imply a vector rather than a scalar